framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -331,35 +362,30 @@ public Range byteRange() {
             return this;
         }
         if (ignoreOverflow) {
-            return new Range(Math.max(from, Byte.MIN_VALUE), Math.min(to, Byte.MAX_VALUE));
+            return create(Math.max(from, Byte.MIN_VALUE), Math.min(to, Byte.MAX_VALUE));
         }
         if (this.isWiderThan(BYTE_WIDTH)) {
             // byte is promoted to int before the operation so no need for explicit casting
             return BYTE_EVERYTHING;
         }
-        byte byteFrom = (byte) this.from;
-        byte byteTo = (byte) this.to;
-        if (byteFrom <= byteTo) {
-            return new Range(byteFrom, byteTo);
-        }
-        return BYTE_EVERYTHING;
+        return createOrElse((byte) this.from, (byte) this.to, BYTE_EVERYTHING);
     }
 
     /**
      * Returns true if the element is contained in this range.
      *
      * @param element the value to seek
-     * @return true if {@code element} is in this range
+     * @return true or false
      */
     public boolean contains(long element) {
         return from <= element && element <= to;
     }
 
     /**
-     * Returns true if the other range is contained in this range.
+     * Returns true if the element is contained in this range.
```

Comment:
Isn't `other range` better than `element` when there is no `element` parameter?

Reply:
corrected